Product/SoapUI_Test/RegressionSuite/Standard/DocQueryFanoutTestTarget/DocQueryFanoutTestTarget-soapui-project.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<con:soapui-project abortOnError="false" name="DocQueryFanoutTestTarget" resourceRoot="${projectDir}" runType="SEQUENTIAL" soapui-version="5.1.2" activeEnvironment="Default" id="820ad3f4-8079-4cb8-a731-9029100b2667" xmlns:con="http://eviware.com/soapui/config">
+<con:soapui-project abortOnError="false" name="DocQueryFanoutTestTarget" resourceRoot="${projectDir}" runType="SEQUENTIAL" soapui-version="4.5.1" activeEnvironment="Default" id="820ad3f4-8079-4cb8-a731-9029100b2667" xmlns:con="http://eviware.com/soapui/config">
```

Comment:
We should settle on a common SoapUI version across the team, preferably in the 5.x family.